sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -17,9 +17,13 @@
 
 package org.apache.spark.sql
 
+import scala.collection.JavaConverters._
```

Comment:
JavaConverters _is_ the explicit one (`.asScala` / `.asJava`); the more implicit one was banned by me in a Scalastyle updtae.